woocommerce/woocommerce-android
# Products m3/add virtual switch

Fixes #2601 by adding the option to display the "Virtual" switch in Product Settings.

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/86316101-8fd5a800-bc49-11ea-8a7d-4e3b84ac9340.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/86316105-919f6b80-bc49-11ea-8e64-392fde03ede9.png" width="300"/>

#### Testing
- Click on a Product from the Products tab.
- Click on the menu option and Product Settings.
- Notice the `Virtual` button displayed. Switch it on/off and click on the `Done` menu button.
- Click on the `Update` button in Product detail and verify that the product is updated successfully.

#### Notes
- This feature is only enabled for debug users and will be released along with the M3 release.
- ~This PR is in draft till this [FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1626) can be reviewed and merged.~

Update release notes:
